desktop_version/src/Game.cpp

```diff
@@ -7606,20 +7606,7 @@ void Game::swnpenalty()
 
 int Game::crewrescued()
 {
-    int temp = 0;
-    if (crewstats[0])
-        temp++;
-    if (crewstats[1])
-        temp++;
-    if (crewstats[2])
-        temp++;
-    if (crewstats[3])
-        temp++;
-    if (crewstats[4])
-        temp++;
-    if (crewstats[5])
-        temp++;
-    return temp;
+    return std::count(crewstats.begin(), crewstats.end(), true);
```

Comment:
Would a `for` loop be more appropriate here? Again, the less STL the better.

Reply:
Fixed in a7a81a45f1c02276dff645a187265352c3b71b18.